### ISSUE TITLE
After search or click on a block reference, I want to see block in context

### ISSUE THREAD

**Issue by Limezy** [athensresearch/athens#1206]
When I search for a block in Athena and click on it, I'll be shown the block in "zoom mode".
I don't know if that's the case for other Athens users here, but I absolutely never find myself in a situation where I want to edit or see that block in zoom mode. I end up dezooming every time, which is quite cumbersome.
Most of the time I want to read again or to find this block in its context.
I think that the standard behaviour should be to show at least the N+1 level, if not the page level.

The same is true for block quotes. When I click on a block quote, most probably I don't do it to edit that block. I do it to find out where that block comes from or read it again in its context. So the same applies : I feel like the standard behaviour should be to show at least the N+1 level, if not the page level of that block.

**Comment by shanberg** [comment]
I think opening the page the block is on and then instantly scrolling to the location of the targeted block should be the default.

Maybe `alt+enter` in Athena opens the block in isolation.

**Comment by Limezy** [comment]
> I think opening the page the block is on and then instantly scrolling to the location of the targeted block should be the default.

Fully agreed

**Comment by Limezy** [comment]
@shanberg but how to deal with closed blocks ?
I would like the following behaviour : open the highest level where the quoted or searched block is directly visible, that is : 
* If the block is in a series of opened blocks, open the page on which the block is and instantly scroll to it
* If the block has a closed block in its parents, zoom in the the closest one and instantly scroll to show it

In other words, show the highest level context we can while keeping the block readable.
Does that make sense ?

**Comment by tangjeff0** [comment]
I think Workflowy/Dynalist have decent interactions for this. Recommend checking em out!

**Comment by sid597** [comment]
Adding for reference #1063 #921
